apps/src/templates/instructions/TopInstructions.story.jsx

```diff
@@ -54,12 +53,6 @@ const createCommonStore = function(options = {}) {
 
     pageConstants.showNextHint = () => {};
     instructionsConstants.noInstructionsWhenCollapsed = false;
-    store.dispatch(
-      setFeedback({
-        message:
-          'some simple, plaintext feedback, used to indicate that something went wrong'
-      })
-    );
```

Comment:
note about this suspicious diff: now that we call `this.getAudioElement()` when `<InlineAudio/>` mounts, anytime we attempt to load a sound that doesn't exist in S3, an error will be thrown (our functionality works as expected and hides the TTS button in this case, but the HTTP error is obviously still logged to the console). however, this was causing our "storybook renders without errors" test to fail. this additional piece of instruction didn't appear to add any additional value to the storybook tests, so i removed it